airootfs/root/.bash_profile

```diff
@@ -1 +1,2 @@
+. ~/.automated_script.sh
```

Comment:
always check for file existance before. this is not python where you can ask for forgivness. this is truly bloody bash